source/helpers/math.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * Linear interpolation. Given a range and a value, returns the
+ * percentage position in the range for the value.
+ *
+ * @param min Lowest value in the range
+ * @param max Highest value in the range
+ * @param interpolation Interpolation amount from 0.0 to 1.0 (or beyond to get a value outside the range [min, max])
+ * @returns Interpolated value
+ */
+export function linearInterpolation(min: number, max: number, interpolation: number): number {
```

Comment:
Why is this function exported?

Reply:
Good catch, I deleted the code that used this. Fixed in latest.